File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;


public class Main {
    public static void main(String[] args) throws IOException{

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int[] arr = new int[9];
        int sum = 0;
        int spy1 = 0, spy2 = 0;

        for(int i = 0; i < 9; i++){
            arr[i] = Integer.parseInt(br.readLine());
            sum += arr[i];
        }

        for(int i = 0; i < 8; i++){
            for(int j = 1; j < 9; j++){
                if(sum - arr[i] - arr[j] == 100){
                    spy1 = arr[i];
                    spy2 = arr[j];
                }

            }
        }

        Arrays.sort(arr);
        for(int i = 0; i < 9; i++){
            if(arr[i] != spy1 && arr[i] != spy2)
            System.out.println(arr[i]);
        }



    }
}

File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.py
import sys
input = sys.stdin.readline

arr = [int(input()) for _ in range(9)]
total = sum(arr)

found = False

for i in range(8):
    for j in range(i + 1, 9):
        if total - arr[i] - arr[j] == 100:
            result = [arr[k] for k in range(9) if k != i and k != j]
            result.sort()
            print(*result, sep="\n")
            found = True
            break
    if found:
        break